packages/vue2/source/pc/qiankun/vue.config.js

```
@@ -0,0 +1,67 @@
+    config.resolve.alias["cloud-ui.vusion"] = "@lcap/pc-ui";
+    /// cloud-ui-alias-end
+  },
+  chainWebpack(config) {
+    // 构建产物中删除console相关代码
+    // config.optimization.minimizer('terser')
+    //     .tap((args) => {
+    //         args[0].terserOptions.compress.drop_console = ['info', 'log', 'warn'];
+    //         return args;
+    //     });
+  },
+  lintOnSave: false,
```

Biome (1.9.4)

[error] This method named chainWebpack is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this method named chainWebpack


(lint/suspicious/noDuplicateObjectKeys)